TWLight/resources/views.py

```diff
@@ -73,28 +73,32 @@ def get_context_data(self, **kwargs):
         If there's no filtering or tags involved, we carry on. Otherwise,
         we add the tag to the context and get the corresponding meta url
         in the template.
-        :param kwargs:
-        :return:
         """
         context = super().get_context_data(**kwargs)
 
         language_code = get_language()
+
         # Changed since T278337: add filter to queryset before we build the partners
         # dictionary
         partner_filtered_list = MainPartnerFilter(
             self.request.GET, queryset=self.get_queryset(), language_code=language_code
         )
         context["filter"] = partner_filtered_list
 
+        # Retrieve the current user and add to context
         user = self.request.user
         if user.is_authenticated:
             user = User.objects.select_related("editor").get(pk=self.request.user.pk)
             context["user"] = user
-            context["editor"] = user.editor
```

Comment:
This didn't seem to be used anywhere. Replaced it with a string and couldn't find the string in page source, and couldn't find `editor` in the relevant html files. Nothing seemed to break - it looks like we're accessing the `editor` via `user.editor` in the html.